### PR TITLE
Direct contributors to the current CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tonycknight**
fixes # _no issue logged, this is a simple spot suggestion_

- [X] I have read the [Contribution Guidelines](../CONTRIBUTING.md)
- [ ] I have commented on the issue above and discussed the intended changes
- [ ] A maintainer has signed off on the changes and the issue was assigned to me
- [ ] All newly added code is adequately covered by tests
- [ ] All existing tests are still running without errors
- [X] The documentation was modified to reflect the changes _OR_ no documentation changes are required.

## Changes

The current PR template contains a link to `CONTRIBUTING.md`. However, when clicking on it, Github redirects to a 404:

image

In short, the URL is missing `/blob/main`. This PR fixes the issue, and so contributors don't have to look too deeply.

### CODE REVIEW

**Review by patriksvensson — APPROVED** [review]

**Comment by patriksvensson** [comment]
Merged! Thank you for your contribution. Much appreciated! 👍
